src/models/SQLCalls.java

```diff
@@ -193,32 +193,44 @@ public void leaveRating(String locationID, String userID, String elevatorRating,
 // --------------------------------- For Profile.java ---------------------------------
 	
 	// sets a user's city
-	public void setCity(String city, String userID) {
+	public String setCity(String username, String city) {
 		try {
 			Statement st = conn.createStatement();
-			st.executeQuery("UPDATE Users SET City='" + city + "' WHERE UserID='" + userID + "'");
+			st.executeQuery("UPDATE Users SET City='" + city + "' WHERE Username='" + username + "'");
 		} catch(SQLException e) {
 			System.out.println("SQLException in setCity: " + e.getMessage());
+			return "SQLException in setCity: " + e.getMessage();
 		}
+		return "success";
```

Comment:
Added return types so the request will either see success or the exception method if an exception has occurred.